common/lib/xmodule/xmodule/course_metadata_utils.py

```diff
@@ -61,10 +61,26 @@ def display_name_with_default(course):
             said course.
     """
     # TODO: Consider changing this to use something like xml.sax.saxutils.escape
+    return course.display_name_unescaped.replace('<', '&lt;').replace('>', '&gt;')
```

Comment:
Ultimately we want #2, but for now we want #1.  You can use the following, which is what the Mako templates use:
from markupsafe import escape